extension-base/src/main/java/com/azure/autorest/extension/base/plugin/JavaSettings.java

```diff
@@ -143,7 +143,8 @@ public static JavaSettings getInstance() {
                 getBooleanValue(host, "use-default-http-status-code-to-exception-type-mapping", false),
                 host.getValue(new TypeReference<Map<Integer, String>>() {}.getType(),
                     "http-status-code-to-exception-type-mapping"),
-                getBooleanValue(host, "partial-update", false)
+                getBooleanValue(host, "partial-update", false),
+                getBooleanValue(host, "generate-live-tests", false)
```

Comment:
Maybe you can use the `generate-tests` option.

Reply:
Makes sense, will do.

Reply:
merged into `generate-tests`